app/src/main/java/app/mywatchlist/screens/MainScreen.kt

```diff
@@ -0,0 +1,69 @@
+package app.mywatchlist
+
+
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.material.BottomNavigation
+import androidx.compose.material.BottomNavigationItem
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.navigation.NavDestination
+import androidx.navigation.NavDestination.Companion.hierarchy
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.compose.rememberNavController
+
+@Composable
+fun MainScreen(){
```

Comment:
This is not really a screen, is it? I'd maybe call it `Navigator` or something and move it to `navigation`. Not that sure about it though. Let me know what you think :)